packages/appstore/src/components/options/options-accounts.scss

```diff
@@ -0,0 +1,205 @@
+.options-container,
+.options-container-app-launcher {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 1.5rem;
+    width: 110.2rem;
```

Comment:
why we need width and height for a wrapper?

Reply:
@mahdiyeh-fs We need width since we are using a scroller inside for accounts container. will remove Height